src/Microsoft.AspNetCore.Sockets.Http/ConnectionManager.cs

```diff
@@ -78,10 +84,10 @@ public DefaultConnectionContext CreateConnection()
 
         public void RemoveConnection(string id)
         {
-            if (_connections.TryRemove(id, out var connection))
+            if (_connections.TryRemove(id, out var pair))
```

Comment:
As above, `if (_connections.TryRemove(id, out (_, timer)))`?

Reply:
why?

Reply:
Destructure looks sweet but IMO not as easy to parse as a reader.